components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/res/Resources.kt

```diff
@@ -1,10 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 package mozilla.components.support.ktx.android.content.res
 
 import android.content.res.Resources
-import android.util.TypedValue
+import mozilla.components.support.ktx.android.util.pxToDp
 
 /**
  * Converts a value in density independent pixels (pxToDp) to the actual pixel values for the display.
  */
-fun Resources.pxToDp(pixels: Int) = TypedValue.applyDimension(
-        TypedValue.COMPLEX_UNIT_DIP, pixels.toFloat(), displayMetrics).toInt()
+fun Resources.pxToDp(pixels: Int) = displayMetrics.pxToDp(pixels)
```

Comment:
Here is the problem. The documentation and the method name do not match. Which one is correct?